static/js/controllers/reports.js

```diff
@@ -369,13 +456,78 @@ var _ = require('underscore'),
         });
       };
       $scope.resetFilterModel = function() {
+        if ($scope.selectMode && $scope.selected && $scope.selected.length) {
+          // can't filter when in select mode
+          return;
+        }
         $scope.filters = {};
         SearchFilters.reset();
         $scope.search();
       };
 
       $scope.search();
 
+      $('.inbox').on('click', '#reports-list .message-wrapper', function(e) {
+        if ($scope.selectMode) {
+          e.preventDefault();
+          e.stopPropagation();
+          var target = $(e.target).closest('li');
+          var reportId = target.attr('data-record-id');
+          var checkbox = target.find('input[type="checkbox"]');
+          var alreadySelected = _.findWhere($scope.selected, { _id: reportId });
+          $timeout(function() {
```

Comment:
Good question. I don't remember. I'll try taking it out. It may have been for debugging purposes.